frontend/src/App.vue

```diff
@@ -1,17 +1,15 @@
 <template>
-  <img
-    class="absolute top-[112px] left-[112px] w-[458px]"
-    src="@/assets/images/logo.svg"
-    alt="logo"
-  />
-  <router-view v-if="config" class="flex-auto m-3.5 z-10" />
+  <div class="flex justify-center items-center py-12">
```

Comment:
I think another breakpoint where decrease the padding of the logo container might be useful for a full hd screen. At the moment a user might still need to scroll

Reply:
I made a re-adjustment to avoid the scrolling and reviewed it with @olympe-silve. So everything is good with the current state